corehq/form_processor/abstract_models.py

```diff
@@ -212,7 +212,8 @@ def resolve_case_property(self, property_name):
         case has more than one parent or grandparent.
         """
         result = []
-        self._resolve_case_property(property_name, result)
+        if isinstance(property_name, basestring):
+            self._resolve_case_property(property_name, result)
```

Comment:
I was just thinking that it could be hard to debug if it accepts anything and just returns an empty list. Probably fine though given what you describe